packaging/manpage.md

```diff
@@ -0,0 +1,39 @@
+% ccx_preCICE(1) 2.17
+% Boris Martin
+% Oct 2021
+
+# NAME
+ccx_preCICE - Adapter for using CalculiX with the preCICE library.
+
+# SYNOPSIS
+**ccx_preCICE** [*OPTION*] *files*
+
+# DESCRIPTION
+
+This page briefly summarizes how to use CalculiX with or without preCICE. For more detailed instructions, read the CalculiX documention on http://www.dhondt.de/ccx_2.17.pdf and preCICE documentation on https://precice.org/docs.html.
+CalculiX is used alone when the -precice-participant option is not used.
+
+## Using with preCICE
+
+## Regular usage
```

Comment:
What is the purpose of these sections and why are they here?

Reply:
I planned to add the part above, but since it's very short these sections should probably be removed